numpyro/distributions/directional.py

```diff
@@ -0,0 +1,62 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+import math
+
+import jax.numpy as jnp
+from jax import lax, dtypes
+
+from numpyro.distributions import constraints
+from numpyro.distributions.distribution import Distribution
+from numpyro.distributions.util import validate_sample
+from numpyro.util import copy_docs_from
+from numpyro.distributions.util import promote_shapes, von_mises_centered
+
+
+@copy_docs_from(Distribution)
+class VonMises(Distribution):
+    arg_constraints = {'loc': constraints.interval(-math.pi, math.pi), 'concentration': constraints.positive}
+
+    @property
+    def support(self):
+        return constraints.interval(-math.pi, math.pi)
```

Comment:
nit: you can do `support = constraints.interval(-math.pi, math.pi)` here

Reply:
Make sense, I'll change it.